openshift/patroni-postgis/docker/init_postgis

```diff
@@ -1,5 +1,12 @@
 
-if [[ (! -z "$APP_USER") &&  (! -z "$APP_PASSWORD") && (! -z "$APP_DATABASE")]]; then
+if [[ (! -z "$APP_USER") &&  (! -z "$APP_PASSWORD") && (! -z "$APP_DATABASE") && (! -z "$NAME")]]; then
+    echo "Initializing archive mode for pgbackrest"
+    psql -c "alter system set archive_mode=on;"
+    psql -c "alter system set archive_command=pgbackrest --stanza=demo-cluster-1 archive-push '%p';"
     echo "Loading PostGIS extensions into $APP_DATABASE"
     psql "$1" -w -c "\c ${APP_DATABASE}" -c "CREATE EXTENSION IF NOT EXISTS postgis;"
-fi
+    echo "Restarting postgres to init config"
+    pg_ctl restart -D /home/postgres/pgdata/pgroot/data
+    echo "Creating pgbackrest stanza"
+    pgbackrest --stanza="$NAME-cluster-1" --pg1-path=/home/postgres/pgdata/pgroot/data --pg1-port=5432 stanza-create
```

Comment:
Wondering where/how the `$NAME` gets set? Does `$NAME` have to be `demo` in order to match the stanza parameter on line 5 above?